### PR TITLE
Content

### PR DESCRIPTION
**Pull request by rajatsharma**

### CODE REVIEW

File: packages/node/src/Client.ts [hunk 1]
```diff
@@ -77,6 +80,19 @@ class Client {
   track(event: string, uid: string, data: Record<string, any>): Promise<Response> {
     return this.makeHttpRequest('track', { event, uid, data })
   }
+
+  content(page: string, options?: ContentOptions) {
```
Comment (RabidFire) [inline packages/node/src/Client.ts:84]:
```
content(
  contentType: ContentOptions['contentType'],
  options?: Omit<ContentOptions, 'contentType'>
)
```

File: packages/node/src/Client.ts [hunk 2]
```diff
@@ -77,6 +80,19 @@ class Client {
...
+    const defaultOptions: Partial<ContentOptions> = { returnType: 'all' }
+
+    if(options) {
```
Comment (RabidFire) [inline packages/node/src/Client.ts:87]: Missing space.

File: packages/node/src/ContentOptionsBuilder.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import type { Response } from 'got'
+
+/* Optional Parameters? */
+export type ContentOptions = {
+  page: string,
```
Comment (RabidFire) [inline packages/node/src/ContentOptionsBuilder.ts:5]: Change `page` to `number`

File: packages/node/src/ContentOptionsBuilder.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  contentType: string,
+  returnType?: 'all' | 'one',
+  filter: Record<string, boolean | string | number>,
+  order: Record<string, 'ASC' | 'DESC'>,
+  limit: number
+}
+
+class ContentOptionsBuilder {
+  private options: Partial<ContentOptions>
```
Comment (RabidFire) [inline packages/node/src/ContentOptionsBuilder.ts:14]: We can remove `Partial` in several places, if we make filter, order, limit, page optional (`?`)

File: packages/node/src/Client.ts (hunk 1 above)
Comment (rajatsharma) [reply packages/node/src/Client.ts]: Removing `contentType` from `ContentOptions`, it'll save us from omits and union types.

**Pull request merged**